ci-jobs/scripts/start-emulator.sh

```diff
@@ -3,8 +3,13 @@
 # This script was copy-pasted from https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/android?view=azure-devops#test-on-the-android-emulator
 # with some changes
 
-# Install AVD files
-declare -r emulator="system-images;android-$ANDROID_SDK_VERSION;default;x86"
+if [ $ANDROID_SDK_VERSION -eq 30 ]; then
+  # It seems chrome browser version update in the test session on system image 25...
+  declare -r emulator="system-images;android-$ANDROID_SDK_VERSION;google_apis;x86"
```

Comment:
API level 30 does not have `default` so far, but potentially this google apps will get chromedriver update.
It would break our chrome stuff test.